README.md

```diff
@@ -35,12 +37,22 @@ To get started with Finch on macOS, the prerequisites are:
 
 Download a release package for your architecture from the [project's GitHub releases](https://github.com/runfinch/finch/releases) page, and once downloaded double click and follow the directions.
 
-#### Installing Finch via [brew](https://brew.sh/)
+##### Installing Finch via [brew](https://brew.sh/)
 
 ```sh
 brew install --cask finch
 ```
 
+#### Windows
+To get started with Finch on Windows, the prerequisites are:
+
+* Windows 10 version 2004 and higher (Build 19041 and higher)
+* Intel system for Windows
```

Comment:
I think "AMD64 based Windows systems" is more accurate since AMD CPUs are also compatible

Reply:
Good point, updated.